x-pack/plugin/enrich/src/main/java/org/elasticsearch/xpack/enrich/rest/RestGenEnrichPolicyAction.java

```diff
@@ -0,0 +1,33 @@
+package org.elasticsearch.xpack.enrich.rest;
+
+import org.elasticsearch.client.node.NodeClient;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.rest.BaseRestHandler;
+import org.elasticsearch.rest.RestController;
+import org.elasticsearch.rest.RestRequest;
+import org.elasticsearch.rest.action.RestToXContentListener;
+import org.elasticsearch.xpack.core.enrich.GetEnrichPolicyAction;
+
+import java.io.IOException;
+
+public class RestGenEnrichPolicyAction extends BaseRestHandler {
+
+    public RestGenEnrichPolicyAction(final Settings settings, final RestController controller) {
+        super(settings);
+        controller.registerHandler(RestRequest.Method.GET, "/{index}/_enrich", this);
```

Comment:
I think we should have the following path structure: `/_enrich/policy/{source_index}`

Reply:
ok, so completely remove the `{index}` and only put it in the payload of the policy itself, right?